src/graph.ts

```diff
@@ -462,6 +465,114 @@ export class FactoryGraph {
         return new Set(Array.from(this.containers).filter((node) => node.item === item))
     }
 
+    /**
+     * Handle the production and transfer of catalysts
+     * */
+    handleCatalysts() {
```

Comment:
Not really an issue, but I would prefer this the be a standalone function which receives the factory as input. I would prefer the `FactoryGraph` to only contain as much logic as really necessary.

Reply:
Fix in https://github.com/lgfrbcsgo/du-factory-generator/commit/5320bc0857710d3f4a1f2e6f9bd056f532d23d9b